config/exports/build-tests.js

```diff
@@ -1,23 +1,30 @@
 // this script watches the tests exported by typescript, copies them to the test directories, and modifies the require("PKG.NAME") statements to test each build
 const cpx = require("cpx");
-const separator = require("path").sep;
+const isWin = /^win/.test(process.platform);
 const Transform = require("stream").Transform;
 const pkg = require('../../package');
-const req = (path) => 'require("' + path + '")';
-const pathUp = (levels) => Array.from(Array(levels), () => '../').join('');
+const req = (modulePath) => 'require("' + modulePath + '")';
+const calculatedSeparator = isWin ? '\\\\' : '/';
```

Comment:
According to the NodeJS [docs](https://nodejs.org/api/path.html#path_path_sep), `path.sep` should already be the platform-specific separator. Why is `calculatedSeparator` necessary?